src/shared-components/optionButton/index.tsx

```diff
@@ -45,19 +66,16 @@ const OptionButton = ({
   </ClickableContainer>
 );
 
+/* eslint-disable react/require-default-props */
```

Comment:
One of the motivations for our more strictly enforcing this rule was to have full documentation of how components were being used. 

As we move to more Function components, which can use ES6 defaults, and especially as we migrate more code over to TypeScript, the usefulness of this rule drops to 0. 

For now, I think it's fine to just in-line disables. I think I will bring up disabling this rule in Demos and Discussions soon.

Reply:
I don't have a strong opinion, but to be honest, I wouldn't hate if it was possible to update the rule so that you either use an ES6 default, required prop, or a default prop. This check has been helpful for me to make sure a component has a consistent contract for what it requires. And if you don't need a prop to be required or have any default value, adding a defaultProp of `undefined` isn't too hard, either.

Reply:
I would be open to ignoring this rule for Function components in this repo: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md#ignorefunctionalcomponents

It would just then be incumbent on us to provide sane defaults, which would be easy enough with TypeScript. But the `eslint` rule isn't refined enough to call out about that.